src/xp.runner/exec/Output.cs

```diff
@@ -19,6 +32,14 @@ public Output()
 
                 Console.CancelKeyPress += (sender, args) => Console.OutputEncoding = original;
                 Console.OutputEncoding = Encoding.UTF8;
+
+                // See "Console Virtual Terminal Sequences"
+                // https://msdn.microsoft.com/en-us/library/windows/desktop/mt638032(v=vs.85).aspx
+                int mode;
+                IntPtr handle = GetStdHandle(STD_OUTPUT_HANDLE);
+                GetConsoleMode(handle, out mode);
+                mode |= ENABLE_VIRTUAL_TERMINAL_PROCESSING;
```

Comment:
https://github.com/iWuzHere/poc-win10ansi/blob/master/main.c uses `ENABLE_VIRTUAL_TERMINAL_PROCESSING | ENABLE_PROCESSED_OUTPUT | ENABLE_WRAP_AT_EOL_OUTPUT`. Do we need the other two, too?